MiniSpace.Services.Reactions/src/MiniSpace.Services.Reactions.Application/Commands/CreateReaction.cs

```diff
@@ -0,0 +1,32 @@
+using System.Data;
+using System.Threading.Tasks.Dataflow;
+using Convey.CQRS.Commands;
+using MiniSpace.Services.Reactions.Core.Entities;
+
+namespace MiniSpace.Services.Reactions.Application.Commands
+{
+    public class CreateReaction : ICommand
+    {
+        public Guid ReactionId {get;}
+        public Guid StudentId { get; }
+        public string StudentFullName {get;}
```

Comment:
We don't need to pass StudentFullName in request, we can get it straight from identity. Have a look here https://github.com/SaintAngeLs/p_software_engineering_2/blob/comments_service/MiniSpace.Services.Comments/src/MiniSpace.Services.Comments.Application/Commands/Handlers/CreateCommentHandler.cs . That also means we shouldn't store student name in StudentRepository